ui/app/templates/components/console/ui-panel.hbs

```diff
@@ -4,9 +4,14 @@
 ~}}
 
 <div class="console-close-button">
-  <button type="button" class="button is-ghost" {{action "closeConsole"}} data-test-console-panel-close>
-    <Icon @name="x" aria-label="Close console" />
-  </button>
+  <Hds::Button
+    class="hds-side-nav__icon-button"
+    {{on "click" (action "closeConsole")}}
+    data-test-console-panel-close
+    @text="Close console"
+    @icon="x"
+    @isIconOnly={{true}}
```

Comment:
Same question here but I doubled back on the docs and wondering if the text is not rendered and only used for accessibility in this context?

Reply:
It mentions it in the Component API section - [docs](https://helios.hashicorp.design/components/button?tab=code#component-api) for quick reference